src/features/prompts/components/prompt-detail.tsx

```diff
@@ -9,77 +9,114 @@ import { api } from "@/src/utils/api";
 import { extractVariables } from "@/src/utils/string";
 import { type Prompt } from "@prisma/client";
 import { Pencil } from "lucide-react";
+import { PromptHistoryNode } from "./prompt-history";
+import { PromotePrompt } from "@/src/features/prompts/components/promote-prompt";
+import { ScrollArea } from "@radix-ui/react-scroll-area";
+import { useQueryParam, NumberParam } from "use-query-params";
 import router from "next/router";
 
 export type PromptDetailProps = {
   projectId: string;
-  promptId: string;
+  promptName: string;
+  //promptVersion: number;
 };
 
 export const PromptDetail = (props: PromptDetailProps) => {
-  const prompt = api.prompts.byId.useQuery({
-    id: props.promptId,
+  const [currentPromptVersion, setCurrentPromptVersion] = useQueryParam(
+    "version",
+    NumberParam,
+  );
+  const prompt = api.prompts.byName.useQuery({
```

Comment:
Why do we need this API call? We could just take the currentPromptVersion and filter the promptHistory, correct? Then we could also delete the backend API.